projects/driver_controls/src/can_fsm.c

```diff
@@ -0,0 +1,42 @@
+#include "can_fsm.h"
+#include "input_event.h"
+#include "event_arbiter.h"
+#include "log.h"
+
+FSM_DECLARE_STATE(state_can_transmit);
+
+FSM_STATE_TRANSITION(state_can_transmit) {
+  FSM_ADD_TRANSITION(INPUT_EVENT_CAN_ID_POWER, state_can_transmit);
+  FSM_ADD_TRANSITION(INPUT_EVENT_CAN_ID_PEDAL, state_can_transmit);
+  FSM_ADD_TRANSITION(INPUT_EVENT_CAN_ID_DIRECTION_SELECTOR, state_can_transmit);
+  FSM_ADD_TRANSITION(INPUT_EVENT_CAN_ID_TURN_SIGNAL, state_can_transmit);
+  FSM_ADD_TRANSITION(INPUT_EVENT_CAN_ID_HAZARD_LIGHT, state_can_transmit);
+  FSM_ADD_TRANSITION(INPUT_EVENT_CAN_ID_MECHANICAL_BRAKE, state_can_transmit);
+}
+
+static StatusCode prv_transmit_data(FSM *fsm, const Event *e, void *context) {
+  InputEventData *data = &e->data;
+
+  printf("Device = %d, State = %d, Data = %d\n",
+          e->id,
+          data->components.state,
+          data->components.data);
+}
+
+static bool prv_check_state(const Event *e) {
```

Comment:
Since you have more than a few filters that just return true, maybe add a default function within `event_arbiter` instead of defining them multiple times? I'd suggest either explicitly adding a no-op/always true function that the FSM should pass in or accept NULL and treat it as a no-op.